backend/vouch/models.py

```diff
@@ -0,0 +1,69 @@
+from django.db import models
+from django.db.models import ObjectDoesNotExist
+from core.models.user import User
+
+# Create your models here.
+class vouch(models.Model):
```

Comment:
We can stick to the Python coding style by using a capital letter.

```suggestion
class Vouch(models.Model)
```